client/client.go

```diff
@@ -223,7 +221,9 @@ func (client *Client) Maintenance() error {
 // the user, and the timestamp of the most recently added warning (useful for
 // silencing the warning alerts, and OKing the returned warnings).
 func (client *Client) WarningsSummary() (count int, timestamp time.Time) {
-	return client.warningCount, client.warningTimestamp
+	// Pebble has never produced warnings (and now it can't), so we can just
```

Comment:
"and now it can't" could be changed to "support code for warnings was removed in July 2024" or "support code for warnings was removed in PR#443" or something, so that casual observers of the future won't have to wonder why it can't produce warnings "now".

This comment also applies to the other instances below where this comment is reproduced.